proxy/src/graphql/project.rs

```diff
@@ -63,12 +63,39 @@ impl From<meta::Project> for Metadata {
     }
 }
 
+/// Shows if a project exists on the Registry and distinguishes between Org and User owned.
+#[derive(GraphQLEnum)]
+pub enum Registered {
+    /// Project is not present on the Registry.
+    Not,
+    /// Project is registered under an Org.
+    Org,
+    /// Project is registered under a User.
+    User,
+}
+
 /// Radicle project for sharing and collaborating.
 #[derive(GraphQLObject)]
-#[graphql(description = "Radicle project")]
+#[graphql(description = "Project")]
 pub struct Project {
     /// Unique identifier of the project in the network.
     pub id: juniper::ID,
     /// Attached metadata, mostly for human pleasure.
     pub metadata: Metadata,
+    /// Signals if a project is on the Registry.
+    pub registered: Registered,
+    /// Coarse set of statistics for the project source code.
+    pub stats: Stats,
+}
+
+/// Coarse statistics for the Project source code.
+#[derive(GraphQLObject)]
+#[graphql(name = "ProjectStats")]
+pub struct Stats {
+    /// Amount of known branches.
+    pub branches: i32,
+    /// Number of commits on the default branch.
+    pub commits: i32,
+    /// Amount of unique commiters on the default branch.
+    pub contributors: i32,
```

Comment:
These stats should actually be dependent on which branch you're on, not the default one.

Reply:
Are their plans to show stats in the context of a branch switched project?

Reply:
Yeah, the project page will show exactly these stats, based on the branch you're on (I thought that's where you got this from)

Reply:
Nope, these stats have been based on the project listing, but this info is valuable. WIll adjust documentation.

Reply:
Ah cool!

Reply:
Actually this is going to be inconsistent. One of the stats is a amount of branches, whcih won't change in a branch context. Treating these as project global stats and look into branch specific stats independently makes more sense.

Reply:
Just for reference, when I wrote down the requirements for this endpoint, I had the user profile project listing in mind. For that screen we don't select a branch, so it would make sense to display global values.

<img width="720" alt="Screenshot 2020-03-02 at 10 11 17" src="https://user-images.githubusercontent.com/158411/75661854-604c6a00-5c6e-11ea-94e1-20bddcbfb438.png">